src/CSET/recipes/example_combined_mask_multiplication.yaml

```diff
@@ -0,0 +1,48 @@
+category: Quick Look
+title: Example adding masks for 'AND' type filters
+description: |
+  Generates and applies two masks, and multiplies them together for more complex stratified analysis.
```

Comment:
Brief description on cases where addition or multiplication of two masks would be used (how would user decide how to choose?)